app/src/App/routes/SurgeRoutes.tsx

```diff
@@ -839,6 +839,160 @@ const surgeCatalogueWash = customWrapRoute({
     },
 });
 
+const surgeCatalogueCMR = customWrapRoute({
```

Comment:
Lets make the variable name consistent with the view name, so that its easier to find the file
```suggestion
const surgeCatalogueOtherCivilMilitaryRelations = customWrapRoute({
```